grapher/controls/CollapsibleList.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react"
+import { CollapsibleList, MoreButton } from "./CollapsibleList"
+import { collapsibleListSampleItems } from "./CollapsibleList.sampleInput"
+
+export default {
+    title: "CollapsibleList",
+    component: CollapsibleList,
+}
+
+export const CollapsibleListComponent = () => {
+    return <CollapsibleList>{collapsibleListSampleItems}</CollapsibleList>
+}
+
+export const MoreButtonComponent = () => {
```

Comment:
Nice! It works.

![Screen Shot 2020-09-10 at 2 49 46 PM](https://user-images.githubusercontent.com/74692/92833801-f2979e00-f374-11ea-9172-6859677dd0a9.png)